### PR TITLE
feat(wrw): add ethw to unsigned-sweep and non-bitgo

### PR DESCRIPTION
**Pull request by wesgraham**
Add support to ETHw to unsigned-sweep and non-bitgo. ETHw is to support an eventual PoW fork of ETH after the merge. Trying to recover it will fail for now since ETHw chain id is not confirmed neither there is a block explorer url.

BG-55419

### CODE REVIEW

**Comment by wesgraham** [comment]
Copied from the already reviewed pr here: https://github.com/BitGo/wallet-recovery-wizard/pull/228 - added chainID to unblock

File: src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx [hunk 1]
```diff
@@ -224,7 +225,7 @@ function Form() {
                     maxPriorityFeePerGas: toWei(maxPriorityFeePerGas),
                   },
                   replayProtectionOptions: {
-                    chain: bitGoEnvironment === 'prod' ? 1 : 5,
+                    chain: coin === 'ethw' ? 10001 : bitGoEnvironment === 'prod' ? 1 : 5,
```
Comment (lifeiscontent) [inline src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx:228]: can we move this to a function that returns the chain? e.g. `getChain(env)`

Comment (lifeiscontent) [reply src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx:228]: also, are we only supporting ethw for prod?

**Review by lifeiscontent — CHANGES_REQUESTED** [review]
chain is getting a little out of hand, lets refactor to a helper

File: src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx (hunk 1 above)
Comment (wesgraham) [reply src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx:228]: yep ethw is just supported for withdrawals in prod!

File: src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx [hunk 2]
```diff
@@ -104,7 +105,7 @@ function Form() {
                     maxPriorityFeePerGas: toWei(maxPriorityFeePerGas),
                   },
                   replayProtectionOptions: {
-                    chain: bitGoEnvironment === 'prod' ? 1 : 5,
+                    chain: coin === 'ethw' ? 10001 : bitGoEnvironment === 'prod' ? 1 : 5,
```
Comment (lifeiscontent) [inline src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx:108]: @wesgraham missed a spot

**Review by lifeiscontent — DISMISSED** [review]

**Review by lifeiscontent — APPROVED** [review]

**Pull request merged**